influxql/engine.go

```diff
@@ -314,7 +314,7 @@ loop:
 	for _, row := range rows {
 		for _, values := range row.Values {
 			t := time.Unix(0, values[0].(int64))
-			values[0] = t.UTC().Format(time.RFC3339Nano)
+			values[0] = t.UTC()
```

Comment:
Yes it's deliberate. The rows shouldn't have string times yet. They should get turned into strings when serialized.